pkg/prometheus/statefulset.go

```diff
@@ -816,9 +819,7 @@ func makeStatefulSetSpec(p monitoringv1.Prometheus, c *Config, ruleConfigMapName
 		if p.Spec.LogFormat != "" {
 			container.Args = append(container.Args, fmt.Sprintf("--log.format=%s", p.Spec.LogFormat))
 		}
-
 		additionalContainers = append(additionalContainers, container)
-		promArgs = append(promArgs, "--storage.tsdb.min-block-duration=2h", "--storage.tsdb.max-block-duration=2h")
```

Comment:
This will remove tsdb compaction settings even when `p.Spec.Thanos.ObjectStorageConfig == nil`.  Do we want that?

Reply:
nvm, I need coffee :smile:

Reply:
=D